modules/nw-multus-ipam-object.adoc

```diff
@@ -192,9 +192,77 @@ spec:
 [id="nw-multus-whereabouts_{context}"]
 == Dynamic IP address assignment configuration with Whereabouts
 
-The Whereabouts CNI plugin allows the dynamic assignment of an IP address to an additional network without the use of a DHCP server.
+The Whereabouts CNI plugin allows the dynamic assignment of an IP address to an additional network without the use of a DHCP server. 
 
-The following table describes the configuration for dynamic IP address assignment with Whereabouts:
+The Whereabouts CNI plugin also supports overlapping IP address ranges and configuration of the same CIDR range multiple times within separate `NetworkAttachmentDefinitions`. This provides greater flexibility and management capabilities in multi-tenant environments.
+
+The following table describes the configuration objects for dynamic IP address assignment with Whereabouts:
+
+.`ipam` whereabouts configuration object
+[cols=".^2,.^2,.^6",options="header"]
+|====
+|Field|Type|Description
+
+|`type`
+|`string`
+|The IPAM address type. The value `whereabouts` is required.
+
+|`range`
+|`string`
+|An IP address and range in CIDR notation. IP addresses are assigned from within this range of addresses.
+
+|`exclude`
+|`array`
+|Optional: A list of zero or more IP addresses and ranges in CIDR notation. IP addresses within an excluded address range are not assigned.
+
+|`network_name`
+|`string`
+| Optional: Helps ensure that each group or domain of pods gets its own set of IP addresses, even if they share the same range of IP addresses. Setting this field is important for keeping networks separate and organized, notably in multi-tenant environments.
+|====
+
+////
+[NOTE]
+=====
+Whereabouts can be used for both IPv4 and IPv6 addresses.
+=====
+////
+
+.Dynamic IP address assignment configuration example that uses Whereabouts
+[source,json]
+----
+{
+  "ipam": {
+    "type": "whereabouts",
+    "range": "192.0.2.192/27",
+    "exclude": [
+       "192.0.2.192/30",
+       "192.0.2.196/32"
+    ]
+  }
+}
+----
+
+.Dynamic IP address assignment configuration examples that use overlapping IP address ranges for multi-tenant networks
+[source,json]
+----
+{
+  "ipam": {
+    "type": "whereabouts",
+    "range": "192.0.2.192/27",
+    "network_name": "<example_a>"
```

Comment:
Maybe this is a standard with the `example` nomenclature and the brackets, and if it is, totally roll with it -- but would it possible to do values of say...

`example_net_blue` and `example_net_green` 

...to show that it's like they're kind of riding two different networks?

Reply:
Got it!